### ISSUE TITLE
Docker `ubuntu-4.3.0.x` or `ubuntu-4.3.0.x-full` cannot start from error `psutil library not found`

### ISSUE THREAD

**Issue by that1matt** [nicolargo/glances#3056]
**Check the bug**
Before filling this bug report, please search if a similar issue already exists.
In this case, just add a comment on this existing issue.

**Describe the bug**
- When using `nicolargo/glances:ubuntu-4.3.0.(1/2/3/latest)-full` or `nicolargo/glances:ubuntu-4.3.0.(1/2/3/latest)` the docker image cannot start, with error `psutil library not found. Glances cannot start.`

**To Reproduce**
1. Run `docker run --rm --name glancestest -v /var/run/docker.sock:/var/run/docker.sock:ro -v /run/user/1000/podman/podman.sock:/run/user/1000/podman/podman.sock:ro --pid host --network host -it nicolargo/glances:ubuntu-4.3.0.1-full`

**Expected behavior**
- Docker container should start without `psutil library not found` error

**Screenshots**

**Environement (please complete the following information)**
 - Operating System (lsb_release -a or OS name/version): `To be completed with result of: lsb_release -a`

```
# Host OS
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 24.04.1 LTS
Release:	24.04
Codename:	noble
```

```
# Docker version
Docker version 27.4.1, build b9d17ea
```

 - Glances & psutil versions: `To be completed with result of: glances -V`

    - Docker tags: `ubuntu-4.3.0.(1/2/3/latest)` or `ubuntu-4.3.0.(1/2/3/latest)-full`

 - How do you install Glances (Pypi package, script, package manager, source): `To be completed`

    - Pulling latest image from Docker hub

 - Glances test: ` To be completed with result of: glances --issue`

    - Unable to run as Docker container will not start

**Additional context**

- Currently version `ubuntu-4.3.0` and `ubuntu-4.3.0-full` work
- Not using a supplied `glances.conf` file.

You can also [pastebin](https://pastebin.com/):

* the Glances configuration file (https://glances.readthedocs.io/en/latest/config.html#location)
* the Glances log file (https://glances.readthedocs.io/en/latest/config.html#logging)

**Comment by rockymountainnative** [comment]
I'm seeing the same issue currently.

**Comment by exelinguy** [comment]
Same issue. Using docker compose.

**Comment by nicolargo** [comment]
Corrected in 4.3.0.4 image.

Sorry for the mess.
